cmake/gadgetron-config.cmake.in

```diff
@@ -1,24 +1,24 @@
 @PACKAGE_INIT@
 
-
-find_package(PythonLibs 3  REQUIRED)
-find_package(Boost 1.65.0 COMPONENTS system program_options filesystem timer REQUIRED )
-if (Boost_VERSION_STRING VERSION_LESS 1.67.0)
-    find_package(Boost 1.65.0 COMPONENTS python3 REQUIRED)
-    set(Boost_PYTHON3_TARGET Boost::python3)
+if (BUILD_PYTHON_SUPPORT)
```

Comment:
Not sure all of this should be excluded if Python is not required. We still need some of the other boost components, right?

Reply:
`find_package(Boost 1.65.0 COMPONENTS system program_options filesystem timer REQUIRED )` has been moved outside the `if(BUILD_PYTHON_SUPPORT)` loop.